packages/origin-ui-core/src/components/OriginConfigurationContext.tsx

```diff
@@ -201,6 +207,7 @@ export function createStyleConfigFromSCSSVariables(scssVariables: any): IOriginS
         PRIMARY_COLOR_DARKER: scssVariables.primaryColorDarker ?? '#894ec5',
         TEXT_COLOR_DEFAULT: scssVariables.textColorDefault ?? '#894ec5',
         BACKGROUND_COLOR_DARK: scssVariables.backgroundColorDark ?? '#894ec5',
+        BACKGROUND_COLOR_DARKER: scssVariables.backgroundColorDarker ?? '#894ec5',
```

Comment:
Why do we keep copying #894ec5? Let's define it once above as a default color and then reference it as a variable.